frontend/public/components/nav/admin-nav.tsx

```diff
@@ -53,7 +53,7 @@ const rolesStartsWith = ['roles', 'clusterroles'];
 const rolebindingsStartsWith = ['rolebindings', 'clusterrolebindings'];
 const quotaStartsWith = ['resourcequotas', 'clusterresourcequotas'];
 const imagestreamsStartsWith = ['imagestreams', 'imagestreamtags'];
-const monitoringAlertsStartsWith = ['monitoring/alerts', 'monitoring/alertrules'];
+const monitoringAlertsStartsWith = ['monitoring/alerts', 'monitoring/alertrules', 'monitoring/silences'];
```

Comment:
Need to add `monitoring/alertmanageryaml` too.

Reply:
fixed